core/src/main/kotlin/io/specmatic/test/ContractTest.kt

```diff
@@ -9,6 +9,7 @@ interface ResponseValidator {
 }
 
 interface ContractTest {
+    val scenario: Scenario
```

Comment:
Seems like the reason to expose this is because we need the scenario metadata and testDescription elsewhere. How about just exposing those methods in ContractTest, so that no-one has to dig in and chain method calls? Then you could remove the `scenario` property.

Reply:
This is a very good idea. I'll do that.

Reply:
Addressed.